tests/tf/systems/madqn_system_test.py

```diff
@@ -44,7 +44,7 @@ def test_madqn_on_debugging_env(self) -> None:
 
         # networks
         network_factory = lp_utils.partial_kwargs(
-            madqn.make_default_networks, value_networks_layer_sizes=(64, 64)
+            madqn.make_default_networks, value_networks_layer_sizes=(32, 32)
```

Comment:
comment applies elsewhere: why were hyparparam changed? to make test faster?

Reply:
Yes, just to make the tests a bit more lightweight. Happy to increase layer sizes and batch sizes if you think that is a better idea?

Reply:
Pros and cons to both but I get the need to make the tests lightweight.